content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,162 @@
+3. Select SAML 2.0 and click **Next**.
+
+![](/img/ws-image0.png)
+
+Provide a name for the SAML application which gets connected to the HPE GreenLake edge-to-cloud platform
+
+![](/img/saml_app-okta.jpg)
+
+**Step 2: How to configure single sign-on settings**
+
+1. Enter the SAML information.
+
+   Under General:
+
+    **Single Sign on URL:** https://sso.common.cloud.hpe.com/sp/ACS.saml2
+
+    **Audience URI (SP Entity ID):** https://sso.common.cloud.hpe.com
+
+    **Name ID format EmailAddress**
+
+    **Application username Email**
+
+    **NameID = user.email**
+
+    **gl_first_name = user.FirstName**
+
+    **gl_last_name = user.LastName**
+
+    **hpe_ccs_attribute = (See Below)**
+
+   See here for IdP attribute details: [](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us)<https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us&page=GUID-D7192971-EF71-4304-B51E-548E7954E644.html>
+
+   A new SAML attribute has been added “hpe_ccs_attribute” which tells HPE GreenLake edge-to-cloud platform and HPE GreenLake Private Cloud Enterprise application the exact role/permissions for each user. The following describes how to format the attribute.
+
+   Format: {version}#{pcid}:{app id}:{role_name}:{ALL_SCOPES}
+
+   N﻿ote :  At present HPE GreenLake Private Cloud Enterprise application role should be excluded.
+
+![](/img/hpe-greenlake-saml-attributes.jpg)
+
+![](/img/workspace-pcid.jpg)
+
+![](/img/glp_role_name.jpg)
+
+The **hpe_ccs_attribute** always starts with version1#. You must first configure the attributes for HPE GreenLake edge-to-cloud platform and To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and ALL_SCOPES**.** 
+
+Example:
+
+version_1#7ede5c36b7b911edacf45a78eb8b07d1:00000000-0000-0000-0000-000000000000:Observer:ALL_SCOPES
+
+![](/img/saml_settings.jpg)
+
+2﻿. Complete the setup.
+
+![](/img/ws-image7.png)
+
+Click Next and Select “Internal App”, then Finish.
+
+**Step 3:** **Export the SAML 2.0 IdP metadata**
+
+1. Click Next – Configure the single sign-on settings
+
+   You will find two options are available: **View Setup Instructions** which steps you through the SAML configuration and **Identity Provider metadata**, which will produce an XML file that can be loaded into HPE GreenLake edge-to-cloud platform application
+
+   Suggestion: Click **Identity Provider metadata** and save the XML data to a file.
+
+   ![](/img/ws-image9.png)
+2. C﻿lick **Next**.       
+3. Select **Internal app**, and click **Finish**.    
+
+   ##### **Step 3.1 :  Access to the SAML application and HPE GreenLake edge-to-cloud platform is determined by assigning only those members or group to the SAML application.**
+
+![](/img/customer-user-assignment-to-saml.jpg)
+
+**S﻿teps to be followed in HPE GreenLake edge-to-cloud platform**
+
+```
```

Comment:
Not sure I understand the reason for this code snippet "Step 4: Create SAML Authorization Profile in the HPE GreenLake edge-to-cloud platform".